src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.professionalapi.generator;
+
+public class ProfessionalApiGeneratorConstants {
+
```

Comment:
why are these java files not in master already? where r u merging this from?

Reply:
Synced master branch

Reply:
removed the file